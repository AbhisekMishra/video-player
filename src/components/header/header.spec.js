import React from 'react';
import { shallow, configure } from 'enzyme';
import Component from './header';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});

describe('Header', () => {
  let props;
  let getShallowComponent;

  beforeEach(() => {
    props = {
        classes: {},
    };

    getShallowComponent = () => shallow(<Component {...props} />).dive();
  });

  describe('render', () => {
    it('should render with all props', () => {
        expect(getShallowComponent().instance()).toMatchSnapshot();
    });
  });
});
