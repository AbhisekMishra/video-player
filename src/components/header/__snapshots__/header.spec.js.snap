// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render should render with all props 1`] = `null`;
